src/main/java/seedu/letsgethired/Main.java

```diff
@@ -34,6 +34,7 @@ public static void main(String[] args) {
         // The warning however, can be safely ignored. Thus, the following log informs
         // the user (if looking at the log output) that the said warning appearing in the log
         // can be ignored.
+        assert false;
```

Comment:
Is this assert statement supposed to be here? 
I think you might have forgotten to remove it after your tutorial.

Reply:
Thanks You Wei! Good Catch 🤣🤣🤣